optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/bestscore/BestScoreProblemStatistic.java

```diff
@@ -73,14 +73,16 @@ public SubSingleStatistic createSubSingleStatistic(SubSingleBenchmarkResult subS
     public void writeGraphFiles(BenchmarkReport benchmarkReport) {
         List<XYPlot> plotList = new ArrayList<XYPlot>(BenchmarkReport.CHARTED_SCORE_LEVEL_SIZE);
         int seriesIndex = 0;
-        for (SingleBenchmarkResult singleBenchmarkResult : problemBenchmarkResult.getSingleBenchmarkResultList()) {
+        List<SingleBenchmarkResult> results = problemBenchmarkResult.getSingleBenchmarkResultList();
+        for (SingleBenchmarkResult singleBenchmarkResult : results) {
```

Comment:
The only advantage is that if I keep it like the original version had it, it won't compile. :-) It has something to do with the new generics I introduced, but I was busy finishing this work and didn't really look into it.